src/main/java/core/basesyntax/service/impl/ProcessServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ProcessService;
+import java.util.Map;
+
+public class ProcessServiceImpl implements ProcessService {
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int FRUIT_TYPE_INDEX = 1;
+    private static final int AMOUNT_INDEX = 2;
+    private static final String COMMA_DELIMITTER = ",";
+    private static final String NEW_LINE_DELIMITER = "\n";
+    private static final String BALANCE_CONSTANT = "b";
+    private static final String SUPPLY_CONSTANT = "s";
+    private static final String RETURN_CONSTANT = "r";
+    private static final String PURCHASE_CONSTANT = "p";
+
+    @Override
+    public StringBuilder getQuantity(String lines, Map<String, Integer> fruitMap) {
```

Comment:
```suggestion
    public Map<> getDataMap(String fileData) {
```
method has one line with stringbuilder and 20 with map

Reply:
fixed